docs/source/onedal/algorithms/statistics/basic-statistics.rst

```diff
@@ -65,6 +65,68 @@ the problem is to compute the following sample characteristics for each feature
    * - Variation coefficient
      - :math:`V(j) = \frac {\text{stdev}(j)} {m(j)}`
 
+.. _basic_statistics_p_math:
+
+Partial Computing
+-----------------
+
+Given a block of a dataset :math:`X = \{ x_1, \ldots, x_n \}` with :math:`n` feature vectors of dimension :math:`p`,
+the sums is a :math:`1 \times p` matrix, the crossproduct is :math:`p \times p` square matrices.
+The sums, the crossproduct are computed with the following formulas:
+
+.. list-table::
+   :widths: 20 60
+   :header-rows: 1
+   :align: left
+
+   * - Statistic
+     - Definition
+   * - Partial Minimum
+     - :math:`min(j) = \smash{\displaystyle \min_i } \{x_{ij}\}`
+   * - Partial Maximum
+     - :math:`max(j) = \smash{\displaystyle \max_i } \{x_{ij}\}`
+   * - Partial Sum
+     - :math:`s(j) = \sum_i x_{ij}`
+   * - Partial Sum of squares
+     - :math:`s_2(j) = \sum_i x_{ij}^2`
+
+.. _basic_statistics_f_math:
+
+Finalize Computing
+------------------
+
+Given a partial result with partial products,
+the means is a :math:`1 \times p` matrix, the covariance and the correlation matrices are :math:`p \times p` square matrices.
```

Comment:
```suggestion
the mean is a :math:`1 \times p` matrix, the covariance and correlation matrices are :math:`p \times p` square matrices.
```

Reply:
Does it make sense to use means, due to here, it means a vector of mean values?

Reply:
yes, in that case, it's fine